sample_data/users.json

```diff
@@ -0,0 +1,177 @@
+[
+  {
+    "_id": "64378abd85008f171cf2990d",
+    "tokenVersion": 0,
+    "appLanguageCode": "en",
+    "createdOrganizations": ["6437904485008f171cf29924"],
+    "createdEvents": [],
+    "userType": "SUPERADMIN",
+    "joinedOrganizations": ["6437904485008f171cf29924"],
+    "registeredEvents": [],
+    "eventAdmin": [],
+    "adminFor": ["6437904485008f171cf29924"],
+    "membershipRequests": [],
+    "organizationsBlockedBy": [],
+    "status": "ACTIVE",
+    "pluginCreationAllowed": true,
+    "adminApproved": true,
+    "firstName": "Test",
```

Comment:
Please give all the users real sounding names. This will be required if we launch a live demo. We need to start thinking this way from early. There should be random first and last name generators online.